ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -376,10 +376,13 @@ def check_backend_asds(logger, disks, backend_name):
                         # check if disk is missing
                         if disk.get('port'):
                             # put object
-                            AlbaCLI.run(command="asd-set",
-                                        named_params={'host': ip_address, 'port': str(disk.get('port')),
-                                                      'long-id': disk.get('asd_id')},
-                                        extra_params=[key, value])
+                            try:
+                                AlbaCLI.run(command="asd-set",
+                                            named_params={'host': ip_address, 'port': str(disk.get('port')),
+                                                          'long-id': disk.get('asd_id')},
+                                            extra_params=[key, value])
+                            except RuntimeError:
+                                raise ConnectionFailedException('Connection failed to disk')
```

Comment:
The only way to differentiate between errors that our wrapper returns is by looking at the supplied message by alba.
For that we would need a list of possible messages that alba returns. Could you provide us that?